docs/migrate/migrate-subgraph.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 2
+title: Migrate from TheGraph
+---
+
+# Migrate from TheGraph
+
+This guide walks through the steps to migrate a subgraph to Subsquid. In what follows we will convert the [Gravatar](https://github.com/graphprotocol/example-subgraph) subgraph into a squid and run it locally. Impatient readers may clone the migrated squid from the `gravatar-squid` branch of the [squid-evm-template repo](https://github.com/subsquid/squid-evm-template):
+
+```bash
+git clone -b gravatar-squid https://github.com/subsquid/squid-evm-template.git
+```
+and run it by following the instructions in README.
+
+Subsquid offers batched processing and a slightly different log handling programming model, which pays off with up to a 100x increase in the indexing speed. At the same time, the concepts of the [schema file](/develop-a-squid/schema-file), code generation and automatic GraphQL API should be familiar for subgraph developers. In most cases the schema file can used by a squid as is. 
+
+## Squid setup 
+
+### 1. Clone template
+
+To begin with, clone the `ethereum-minimal` template from the [squid-evm-template repo](https://github.com/subsquid/squid-evm-template):
```

Comment:
Is this going to be changed to https://github.com/subsquid/squid-ethereum-template/ ?